django/db/backends/sqlite3/features.py

```diff
@@ -32,6 +32,8 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     # Is "ALTER TABLE ... DROP COLUMN" supported?
     can_alter_table_drop_column = Database.sqlite_version_info >= (3, 35, 5)
     supports_parentheses_in_compound = False
+    # NULLS LAST/FIRST emulation on < 3.30 requires subquery wrapping.
+    requires_compound_order_by_subquery = Database.sqlite_version_info >= (3, 30, 0)
```

Comment:
We can reuse `supports_order_by_nulls_modifier`:
```suggestion
    requires_compound_order_by_subquery = property(operator.attrgetter("supports_order_by_nulls_modifier"))
```

Reply:
yeah that works for me!

Reply:
OK we cannot reuse it because it should be `< (3, 30)` not `>= (3, 30, 0)`.

Reply:
Fixed.